go.mod

```
@@ -1,12 +1,19 @@
 module github.com/eteu-technologies/near-api-go
 
-go 1.16
+go 1.17
```

```suggestion
go 1.18
```

Might as well bump to 1.18 now.

---

This is just to upgrade the version of go by the way. As for 1.17 or 1.18, it has no essential impact.

---

Yeah so let's bump it to 1.18 then.